types/index.d.ts

```diff
@@ -0,0 +1,90 @@
+import Vue, { PluginFunction, VueConstructor } from "vue"
```

Comment:
This library exports a plugin object, not a plugin function. As such:

```suggestion
import Vue, { PluginObject, VueConstructor } from "vue"

declare const VueGtm: VueGtmPlugin
export default VueGtm
export interface VueGtmPlugin extends PluginObject<VueGtmUseOptions> {
}
```